app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class KeywordsController < ApplicationController
+      include ::Pagy::Backend
+      include API::V1::Pagy::JSONAPIConcern
+      include ErrorHandlerConcern
+
+      def index
+        pagy, keywords_list = pagy(keywords)
+
+        if keywords.any?
+          render json: KeywordSerializer.new(keywords_list, pagy_options(pagy)).serializable_hash, status: :ok
+        else
+          render_empty
+        end
+      rescue ::Pagy::OverflowError => e
```

Comment:
since this could be common error among several resources, it could good to add it into application_controller

```
    rescue_from Pagy::OverflowError, ::Pagy::VariableError => e
        render_error(details: message, source: error.variable, status: :unprocessable_entity)
    end
```

Reply:
Nice! :) 
Done ;-)